os.bpmsuite.businesscentral/added/launch/bpmsuite-businesscentral.sh

```diff
@@ -119,3 +127,33 @@ function configure_misc_security() {
 function configure_metaspace() {
     export GC_MAX_METASPACE_SIZE=${WORKBENCH_MAX_METASPACE_SIZE:-1024}
 }
+
+# required envs for HA
+function haEnabled() {
```

Comment:
I only see this function used in one place. Is it being called from other scripts as well? If not, I think it should go in the configureHA function.